### PR TITLE
Update Transformer.jl

### PR DESCRIPTION
**Pull request by luap-pik**
correct function signature

### CODE REVIEW

File: src/lines/Transformer.jl [hunk 1]
```diff
@@ -1,15 +1,32 @@
...
+- `to` : end node
+- `y`: transformer admittance
+- `t_ratio`: transformation ration
+
+# Assumptions
+
 The admittance is here taken to be on the high-voltage side.
 """
-@Line Transformer(y, t_ratio) begin
+@Line Transformer(from, to, y, t_ratio) begin
```
Comment (SabineAuer) [inline src/lines/Transformer.jl:26]: @luap-pik: why did you add the "from" and "to" here when it is unused?

**Comment by SabineAuer** [comment]
@luap-pik: do you have an idea for a test for the transformer? Since we do not use it much, yet, I think we do not realize when we break something here.

File: src/lines/Transformer.jl (hunk 1 above)
Comment (luap-pik) [reply src/lines/Transformer.jl:26]: Oh it was just annoying in my application that transformers don't follow the way other lines are used. For instance, when I want to do sth like "Give me the voltage  at the `from` node" in a control loop, I need (sketch)

```Julia
...
trafo_idx = findfirst(pg.lines isa Transformer)
from_node = pg.lines[trafo_idx].from
voltage = operationpoint[from_node, :v]
...
```

**Comment by luap-pik** [comment]
good point, there were actually no tests yet. I added a test file.

**Comment by SabineAuer** [comment]
Thanks a lot for adding the test. And I understand now what is the point of the update. I will merge the PR now. I will just commit the merge from the master before and run the tests again.
